static/sass/styles.scss

```diff
@@ -1369,6 +1369,9 @@ hr.p-separator.is-shallow {
   }
 }
 
+.is-paper--input {
+  background-color: rgba(0, 0, 0, 0.04);
+  
```

Comment:
CSS is broken.

Reply:
It might fix all the failing pipelines and if design is happy then we can try an merge it today.